glad/files/gl.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The change should also work with the BOM, since the files are just dumped directly from the git without any (manual) processing.

I tested it locally and it seems to work fine with the BOM, can you revert this change?